HTTPServerProject/Program.cs

```diff
@@ -22,7 +24,7 @@ public Server(TcpClient tcpc)
         public void Conversation()
         {
             NetworkStream stream = client.GetStream();
```

Comment:
Remember all of these can be `var` - `var stream = client.GetStream()`, `var writer = new MyStreamWriter(stream)` etc. It's less to type and makes your code more flexible (less places to change names when you change a variable).

Reply:
updated for this